sites/website/src/docs/resources/faq.md

```diff
@@ -27,4 +27,11 @@ At this time, `fast-element` has a focus that's a bit different from the typical
 
 ### Are Web Components "done"?
 
-The work on Web Component standards, like the rest of the web, is ongoing. New APIs continue to be designed and released. Some recent APIs include Form Associated Custom Element APIs and CSS Shadow Parts. The W3C is currently working on standards for things like Constructible Style Sheets, Declarative Shadow DOM, Scoped Element Registries, Custom Pseudo Selectors, and more.
\ No newline at end of file
+The work on Web Component standards, like the rest of the web, is ongoing. New APIs continue to be designed and released. Some recent APIs include Form Associated Custom Element APIs and CSS Shadow Parts. The W3C is currently working on standards for things like Constructible Style Sheets, Declarative Shadow DOM, Scoped Element Registries, Custom Pseudo Selectors, and more.
```

Comment:
Would it be useful to link each of the W3C standards to it's location for those that would like to learn more.  Backlinks also help with SEO.

Reply:
I didn't want to get into maintaining a list of components here so just linked Open UI. I just pushed a commit that links to our specs folder as well, which I think would be helpful for folks who want to dig in deeper.